### PR TITLE
feat: add terms & conditions page

### PR DESCRIPTION
**Pull request by lcsfelix**

### CODE REVIEW

**Comment by lcsfelix** [comment]
[Create the terms and conditions screen](https://trello.com/c/Wa0lscuJ/153-create-the-terms-and-conditions-screen)

File: new-lamassu-admin/src/pages/OperatorInfo/OperatorInfo.js [hunk 1]
```diff
@@ -28,7 +29,8 @@ const TERMS_CONDITIONS = 'Terms & Conditions'
 const pages = [CONTACT_INFORMATION, RECEIPT, COIN_ATM_RADAR, TERMS_CONDITIONS]
 
 const OperatorInfo = () => {
-  const [selected, setSelected] = useState(CONTACT_INFORMATION)
+  // const [selected, setSelected] = useState(CONTACT_INFORMATION)
```
Comment (RafaelTaranto) [inline new-lamassu-admin/src/pages/OperatorInfo/OperatorInfo.js]: Why is this commented out?

File: new-lamassu-admin/src/pages/OperatorInfo/TermsConditions.js [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+            screenTitle: findValue('screenTitle'),
+            textContent: findValue('textContent'),
+            acceptButtonText: findValue('acceptButtonText'),
+            cancelButtonText: findValue('cancelButtonText')
+          }}
+          validationSchema={Yup.object().shape({
+            screenTitle: Yup.string().max(50, 'Too long'),
+            acceptButtonText: Yup.string().max(15, 'Too long'),
+            cancelButtonText: Yup.string().max(15, 'Too long')
+          })}
```
Comment (RafaelTaranto) [inline new-lamassu-admin/src/pages/OperatorInfo/TermsConditions.js]: Move this to their own objects.

File: new-lamassu-admin/src/pages/OperatorInfo/TermsConditions.js [hunk 3]
```diff
@@ -0,0 +1,166 @@
...
+      variables: { config: { termsAndConditions: it } }
+    })
+  }
+
+  const handleEnable = () => {
+    const s = !showOnScreen
+    save({ show: s })
+  }
+
+  const classes = useStyles()
```
Comment (RafaelTaranto) [inline new-lamassu-admin/src/pages/OperatorInfo/TermsConditions.js]: Move this to the top of the Component. Generally hooks should be the first thing called within the component function.

**Review by RafaelTaranto — CHANGES_REQUESTED** [review]

**Review by RafaelTaranto — APPROVED** [review]

**Comment by joshmh** [comment]
After merging some other PRs this PR now has conflicts. @lcsfelix can you resolve them?

**Pull request merged**